### ISSUE TITLE
Line and Column not listed

### ISSUE THREAD

**Issue by pmcoz** [kantlove/vscode-todo-parser#17]
Unlike in the documentation animation, the output pane doesn't list the line and column of TODO items. Probably as a consequenceparse, Ctrl-clicking on each item does nothing. I've tried with .js and .java.

`1.`
`From c:\Users\Peter M\Documents\Development\Web\apache\concrete5_dirty\application\blocks\gond_read_more_bottom\temp.java`
`----------------------------------`
`todo: finish me`
` `
`==================================`
`Found 1 TODO.`

I apologise if I'm just doing something dumb!

**Comment by kantlove** [comment]
Thanks for reporting :+1: 

The line and column are buggy so I temporarily removed them recently (they will be back soon!)
Can you try Ctrl+Click on the file name? Because Vscode is supposed to turn that into a hyperlink.

**Comment by pmcoz** [comment]
Oh, sorry! Then it's working as intended. :)

Yes, ctrl-click opens the correct file pane, but doesn't go to the right line (as you know).

Keep up the great work! This is a really useful extension.

**Comment by kantlove** [comment]
Well, I really want it to go to the correct line but still not found the right API for that yet :disappointed:

**Comment by pmcoz** [comment]
I don't know if this helps, but I found evidence of [cursorMove](https://code.visualstudio.com/docs/extensionAPI/vscode-api-commands) and [moveTo](https://github.com/Microsoft/vscode/commit/35d592de032f031f791745ec9fed80fd8e8712ae). I have no idea how you'd use them.

**Comment by kantlove** [comment]
Hey that's cool! This will definitely help :+1:

**Comment by kantlove** [comment]
With [v1.7.1](https://github.com/kantlove/vscode-todo-parser/releases/tag/v1.7.1), you can now Ctrl+Click on the file name to jump to the TODO.

**Comment by pmcoz** [comment]
Thanks; that was quick! Unfortunately, it seems to detect the wrong line number (slightly too high) sometimes, but I guess that's a separate issue.

**Comment by kantlove** [comment]
Can you provide me with an example?

**Comment by pmcoz** [comment]
I had to rename this to upload: it's just .js

Comment is on line 382, but is reported to be on line 395. I initially wondered if the problem was something to do with folding (which I use a lot), but it doesn't seem to be.

I tried quickly to track down exactly where the problem starts; it's around line 10. I can't see anything distinctive in that vicinity.

Good luck!
[auto.js.txt](https://github.com/kantlove/vscode-todo-parser/files/426091/auto.js.txt)
